### PR TITLE
Add Picture Page

### LINKED ISSUE #33: Connect "Picture"- Page to Database
- [ ] #7 
- [ ] #2 
- [ ] Connect "Picture"- Page to Database

### LINKED ISSUE #37: Research picture uploading options

### PR DESCRIPTION
**Pull request by lauravikanis**
Add pictures to locationPage 
close #37  close #33

### CODE REVIEW

File: client/src/pages/Location.js [hunk 1]
```diff
@@ -40,6 +39,8 @@ const Location = () => {
   if (error) {
     return `Ein Fehler ist aufgetreten: ${error.message}`;
   }
+  const locationName = name;
+  console.log(locationName);
```
Comment (Sina-Sche) [inline client/src/pages/Location.js:43]: do you still need the console.log here? 
```suggestion
 
```

File: client/src/pages/Pictures.js [hunk 2]
```diff
@@ -15,20 +20,37 @@ const PicturesDiv = styled.div`
 `;
 
 const Pictures = () => {
+  const locationName = useParams();
+  console.log(locationName.name);
```
Comment (Sina-Sche) [inline client/src/pages/Pictures.js:24]:
```suggestion

```

you can probably remove the console.log here

**Review by Sina-Sche — COMMENTED** [review]
lgtm 🚀

**Comment by lauravikanis** [comment]
> lgtm 🚀

Thanks for your review. I deleted all the logs. Maybe you want to review the code again because I change quite a lot after the last review :)

**Review by mariegross — APPROVED** [review]
LGTM :)

**Pull request merged**